lib/suspenders/generators/app_generator.rb

```diff
@@ -95,14 +95,8 @@ def setup_development_environment
 
     def setup_test_environment
       say 'Setting up the test environment'
-      build :generate_rspec
-      build :configure_rspec
       build :configure_background_jobs_for_rspec
```

Comment:
Out of curiosity, why not move these into the generator as well? Will they be part of a future move?

Reply:
I was thinking that the `configure_background_jobs_for_rspec` would go into a `suspenders:jobs` generator. `configure_ci` is in `suspenders:ci` and `configure_capybara_webkit` is in `suspenders:js_driver`.

Reply:
Sounds right to me!